src/main/java/com/aws/greengrass/clientdevices/auth/certificate/handlers/CACertificateChainChangedHandler.java

```diff
@@ -43,7 +44,10 @@ public void listen() {
      */
     @Override
     public void accept(CACertificateChainChanged event) {
-        // TODO: Move retry logic from the domain to here, based on the result returned from the useCase
-        useCases.get(RegisterCertificateAuthorityUseCase.class).apply(null);
+        try {
+            useCases.get(RegisterCertificateAuthorityUseCase.class).apply(null);
+        } catch (UseCaseException e) {
+            // TODO: Move retry logic from the domain to here, based on the result returned from the useCase
```

Comment:
We should have a backlog item to track this if we don't currently have a mechanism to retry failures. At the very least, do we need to log here?